### ISSUE TITLE
warm up epochs in 400-E pretraining

### ISSUE THREAD

**Issue by jimmy-dq** [junchen14/LoMaR#3]
Hi JunChen,
Thanks for your great work. In this repos, you show the running command as belowfor 400-E training:
python -m torch.distributed.launch --nproc_per_node=4 --nnodes=1 \
--master_addr=127.0.0.1 --master_port=29517 main_pretrain_lomar.py \
    --batch_size 256 \
    --accum_iter 4 \
    --output_dir ${LOG_DIR} \
    --log_dir ${LOG_DIR} \
    --model mae_vit_base_patch16 \
    --norm_pix_loss \
    --distributed \
    --epochs 400 \
    --warmup_epochs 20 \
    --blr 1.5e-4 --weight_decay 0.05 \
    --window_size 7 \
    --num_window 4 \
    --mask_ratio 0.8 \
    --data_path ${IMAGENET_DIR}
Here 20 epochs are used for warm up. But in your supplementary, you mentioned that 10 for pretraining 400 epochs. What's the exact setting to reproduce the results in Table-1? Does the number of warm up epoch severly affect the final performance?
Many thanks for your time.

**Comment by junchen14** [comment]
sorry for this inconsistency. It should be 20 epochs for warm up.  We treat first 5% epochs as the warm-up.  
We did not ablate much on how the number of warm up epoch affects the final performance.

**Comment by jimmy-dq** [comment]
Got it. Thanks for the clarification.
